src/main/scala/li/cil/oc/integration/appeng/DriverBlockInterface.scala

```diff
@@ -162,6 +162,7 @@ object DriverBlockInterface extends DriverSidedTileEntity {
         val nbt = new NBTTagCompound()
         stack.writeToNBT(nbt)
         nbt.setInteger("Count", stack.stackSize)
+        nbt.setLong("Cnt", stack.stackSize)
```

Comment:
I hope AC2FC won't take item for fluid in that case...

Reply:
I tested it and it seemed to work fine, AE2FC checks if the item is a drop for the fluid part.

Reply:
I'd still do
```
val tag = inTag.getCompoundTagAt(index)
if (tag.getLong("Cnt") > 0)
  nbt.setLong("Cnt", stack.stackSize)
else
  nbt.setLong("Count", stack.stackSize)
```
just to be safe an mimick the logic for `get`.

I'd also consider just replacing this whole part with 
```
if (stack != null) {
  val nbt = inTag.getCompoundTagAt(index)
  if (nbt.getLong("Cnt") > 0)
    nbt.setLong("Cnt", stack.stackSize)
  else
    nbt.setLong("Count", stack.stackSize)
}
```
to preserve other tags in case there are any in the future. Though I can't say for certain if updating NBT in-place will register properly.

Reply:
I tried a few ways to avoid setting Count if the pattern is a fluid pattern, but it always invalidated the pattern. Having both Count and Cnt set doesn't impact functionality (tested on full pack), just introduces a small amout of extra NBT to the pattern if it is touched by OC.

Reply:
"Count" is present but set to 0 for every fluid pattern normally